### PR TITLE
screencap: add pt_BR translation

### PR DESCRIPTION
**Pull request by FvFurquim**
- [ X ] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [ X ] The page(s) have at most 8 examples.
- [ X ] The page description(s) have links to documentation or a homepage.
- [ X ] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [ X ] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).

### CODE REVIEW

File: pages.pt_BR/android/screencap.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# screencap
+
+> Tira uma scrennshot do display mobile.
+> Este comando apenas pode ser usado atraves de `adb shell`.
+> Mais informacoes: <https://developer.android.com/studio/command-line/adb#screencap>.
+
+- Tira uma screenshot:
+
+`adb shell screencap {{path/to/file}}`
```
Comment (navarroaxel) [inline pages.pt_BR/android/screencap.md:9]: Oh, this is an error. if this page is on `android` platform. Please apply this fix to the screencap pages.
```suggestion
`screencap {{path/to/file}}`
```

Comment (FvFurquim) [reply pages.pt_BR/android/screencap.md]: Ok, I'll fix it!

**Review by marchersimon — APPROVED** [review]
Thanks @FvFurquim

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**